agent/util-scripts/pbench-sysinfo-dump

```diff
@@ -127,6 +127,45 @@ function collect_ara_data {
 	fi
 }
 
+function collect_stockpile_data {
+	# parse config
+	stockpile_path=$(getconf.py stockpile_path stockpile)
+	stockpile_user=$(getconf.py stockpile_user stockpile)
+	stockpile_local_remote_user=$(getconf.py local_remote_user stockpile)
+	stockpile_host_remote_user=$(getconf.py host_remote_user stockpile)
+	stockpile_output_path=$(getconf.py stockpile_output_path stockpile)
+	stockpile_log=$(getconf.py stockpile_log stockpile)
+
+	if [[ -z "$stockpile_path" ]]; then
+		stockpile_path=$(getconf.py pbench_install_dir DEFAULT)
+	fi
+	if [[ -d "$stockpile_path/stockpile" ]]; then
+        	rm -rf $stockpile_path/stockpile
```

Comment:
Why is this removed? The playbook is removed along with the directory and the collection fails.

Reply:
Looks like this was resolved.